frontend/pages/views/PhotographerListView.js

```diff
@@ -83,30 +106,72 @@ export class PhotographerListView extends React.Component {
 
         this.state = {
             timer: null,
+            searchQueries: {name: "", location: "", square: "", nameStartsWith: ""},
             activeDropdown: null,
             photographers: [],
+            pageNumber: 1,
+            isLastPage: false,
+            sortBy: "",
+            dropdownSearchOptions: initialDropdownOptions,
         };
+        this.refetchPhotographers = this.refetchPhotographers.bind(this);
+        this.loadDropdownSearchOptions = this.loadDropdownSearchOptions.bind(this);
+        this.resetPaginationParameters = this.resetPaginationParameters.bind(this);
+        this.handleScroll = this.handleScroll.bind(this);
+    }
 
-
-        // THIS IS DUMMY DATA
-        this.LOCATIONS = ["1", "2", "3"];
-        this.SQUARES = ["4", "5", "6"];
-        this.ALPHABET = ["7", "8", "9"];
-        this.SORTS = ["10", "11", "12"];
+    async loadDropdownSearchOptions() {
+        const fetchOptions = async () => {
+            try {
+                const res = await fetch("/api/search_photographers/dropdown_options");
+                return res.json();
+            } catch {
+                console.log("ERROR FETCHING OPTIONS FOR DROPDOWN SEARCH");
+                return initialDropdownOptions;
+            }
+        };
+        const searchOptions = await fetchOptions();
+        this.setState({dropdownSearchOptions: searchOptions});
     }
 
-    updatePhotographers(name) {
-        const fetchPhotographers = async (sq) => {
+    refetchPhotographers() {
+        const fetchPhotographers = async (searchQueries, orderBy) => {
+            if (this.state.isLastPage) {
+                return;
+            }
+            const {name, nameStartsWith, location, square} = searchQueries;
+            const newPageNumber = this.state.pageNumber + 1;
             try {
-                const res = await fetch(`/api/search_photographers?name=${sq}`);
+                const res = await fetch(
+                    `/api/search_photographers?
+						name=${!name ? "" : name}&
```

Comment:
The default values here are already empty strings (in the constructor), so no need to set empty strings here -- just use the values directly.